Maui/MLTrainer/MLTrainer.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net6.0-maccatalyst</TargetFramework>
-		<TargetFramework Condition="$([MSBuild]::IsOSPlatform('windows'))">net6.0-windows10.0.19041.0</TargetFramework>
+		<TargetFrameworks>net6.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">net6.0-windows10.0.19041.0</TargetFrameworks>
```

Comment:
I think you'll have to update the GitHub action:
```
Error: /Users/runner/.dotnet/sdk/6.0.401/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.CrossTargeting.targets(27,5): error NETSDK1129: The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, you must specify the framework for the published application. [/Users/runner/work/inclusive-code-reviews-ml/inclusive-code-reviews-ml/Maui/MLTrainer/MLTrainer.csproj]
Error: Process completed with exit code 1.
```

This line would need `-f net6.0-maccatalyst` or `-f net6.0-windows10.0.19041.0` now?

https://github.com/jonathanpeppers/inclusive-code-reviews-ml/blob/ed892384c8f66b4785c034da86f62c1a6b715caa/.github/workflows/maui.yml#L42

You might be able to make a variable to put the right value per platform, but I don't have an example of a GitHub action that already does this.

Reply:
Note to self from Peppers: `Condition=" '$(BuildingInsideVisualStudio)' == 'true' "` may identify if we are in a Mac Build or `Condition="$([MSBuild]::IsOSPlatform('osx'))"`